feat: Add MultiSelect widget to chatmark library

yangbobo2021 opened the pull request:
This pull request adds a new MultiSelect widget to the chatmark library. The changes include:

- Implementing the MultiSelect class in widgets.py
- Updating __init__.py to export MultiSelect
- Adding support for multiple option selection with checkboxes

These additions will enhance the functionality of the chatmark library by providing users with a multi-select option capability.

yangbobo2021 committed:
feat: Add MultiSelect widget to chatmark library

- Implement MultiSelect class in widgets.py
- Update __init__.py to export MultiSelect
- Add support for multiple option selection with checkboxes

kagami-l commented on lib/chatmark/widgets.py:
```diff
@@ -162,6 +162,94 @@ def _parse_response(self, response: Dict):
 
         self._selections = selections
 
+class multiSelect(Widget):
```
类名首字母要大写

然后python这边只是chatmark语法里的符号不一样、其他逻辑一样，可以写成CheckBox的子类、只重写_in_chatmark函数

yangbobo2021 committed:
feat: Refactor MultiSelect to inherit from Checkbox

- Rename multiSelect class to MultiSelect for consistency
- Inherit MultiSelect from Checkbox to reduce code duplication
- Remove redundant properties and methods in MultiSelect class

yangbobo2021 committed:
feat: Add MultiSelect widget and update imports

- Implement MultiSelect widget inheriting from Checkbox
- Update import order in __init__.py for consistency
- Prepare groundwork for multi-select dropdown functionality
